e2e/test/scenarios/dashboard-cards/click-behavior.cy.spec.js

```diff
@@ -215,7 +215,7 @@ describe("scenarios > dashboard > dashboard cards > click behavior", () => {
       cy.get("aside").findByText("No available targets").should("exist");
       cy.get("aside").button("Done").click();
 
-      saveDashboard();
+      saveDashboard({ waitMs: 250 });
```

Comment:
This is needed for [the same reason](https://github.com/metabase/metabase/pull/36910/files#r1437018691) it was needed before.